Dockerfile

```diff
@@ -8,4 +8,5 @@ RUN export GOPATH=$(go env GOPATH) && \
 COPY . $GOPATH/src/github.com/percona/pmm/
 WORKDIR $GOPATH/src/github.com/percona/pmm/api-tests/
 
-CMD make init run-race
+CMD ["make", "init", "run-race"]
```

Comment:
This allows to get rid of the docker build warning that says that it's better to use `exec` syntax vs `bash` syntax for RUN, CMD and ENTRYPOINT directives.